updating SOP (#185)
CDCgov/cfa-epinow2-pipeline: SOP.md

## Epinow2 Rt Pipeline Local Run
This document is meant to guide someone in running the weekly Rt estimation pipeline from within the VAP (Virtual Analytic Platform). The main command for running the weekly pipeline (found in the Makefile) is `make run-batch`. Running this will utilize a configuration file (.json) specified from within the associated blob storage account and will produce outputs in the `rt-epinow2-output` Azure blob storage account.

### Pre-requisites
1. VAP environment & Account
	- git (`sudo apt-get install git`)
	- docker CLI
	- gh CLI (`sudo apt-get install gh`)

2. cfa-epinow2-pipeline repository in VAP
	- Navigate to where you would like to clone the repository code

> 	- Clone the repository (`git clone https://www.github.com/cdcgov/cfa-epinow2-pipeline`)

Reviewer: Probably easier to make it

```sh
gh auth login
gh repo clone cdcgov/cfa-epinow2-pipeline
```

Then `gh` will handle your SSO token

Author: Sure, I could add the podman section. Were you thinking, just at the end add of the SOP add a note on it?

Reviewer: Yes!

3. Authentication to Azure

> To authenticate to the requisite Azure resources to source a .env file containing a service principal or other credentials necessary for authentication.

Reviewer: ```suggestion
To authenticate to the requisite Azure resources provide a `.env` file containing the secrets necessary for authentication.
```

	- Request access to necessary Azure credential file (.env) from any of the admins listed in the README.md
	- decrypt the file (`gpg --decrypt .env`)

> 	- source it into your environemnt (`source .env`)

Reviewer: ```suggestion
	- Place the decrypted file in your `cfa-epinow2-pipeline` directory
```

You don't need to source it into _your_ environment. It needs to be accessible for _Docker_ to source into _Docker's_ environment.

### Test Pre-requisites are Setup
#### Test Configuration Generation
1. `make config`
Running this command runs code located in the CDCgov/cfa-config-generator repository. This command creates a configuration file and saves it into the appropriate azure blob storage account.
If you receive an error that you do not have the necessary permissions to run this command please reach out Agastya Mondal (ab59@cdc.gov) for assistance

#### Test make run command
1. The following command will test your setup for using the `CFAEpiNow2Pipeline` package as well as your connection to the azure resources

>  `make run CONFIG=test/test.json`

Reviewer: Did you ever get this working?

Author: I did! Saving the .json in the base directory of blob won't work. So `make run CONFIG=test.json` won't work (even if the file is there). So I just saved it into a test folder as well and now it works.
